app/Database/Migrations/2024-10-07-172154_AddTeamMember.php

```diff
@@ -14,7 +14,16 @@ public function up(): void
         $speakerTableFields = $this->db->getFieldData('Speaker');
 
         $fields = [];
+        $fields['id'] = [
+            'type' => 'INT',
+            'unsigned' => true,
+            'auto_increment' => true,
+        ];
+
         foreach ($speakerTableFields as $field) {
+            if ($field->name === 'id') {
+                continue;
+            }
```

Comment:
I cannot comment on the correct line (since it's not modified in the PR), but I think we can remove line 33 (`'auto_increment' => $field->name === 'id',`), just like in the `AddMediaPartner` migration.